osversion/osversion_windows_test.go

```diff
@@ -0,0 +1,20 @@
+package osversion
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestOSVersionString(t *testing.T) {
+	v := OSVersion{
+		Version:      123,
```

Comment:
Technically, this should be `809042555` (`123 | (2 << 8) | (12345 << 16)`), but I think you can just set it to zero for this test

Reply:
Ah, good one; yes I can set it to `809042555` (I wanted to avoid setting it to `0`, as that's the default value, and such values _could_ e.g. be ignored / omit-empty-like things).

Reply:
Updated 👍 ptal